docs/_template.html

```diff
@@ -2,84 +2,102 @@
+                <a class="nav-link active" aria-current="page" href="index.html">Docs</a>
+                <a class="nav-link" href="#">Tools</a>
             </div>
+            <form class="d-flex">
+              <div class="input-group me-4">
+              <label for="search-by">
+                <i class="fas fa-search"></i>
+              </label>
+                <div class="btn border-end-0 border">
+                  <i class="bi bi-search"></i>
+                </div>
+                <input aria-label="Search" class="form-control  border-start-0 border" data-search-input=""
+                  id="search-by" type="search" placeholder="Search..." />
+                <span class="input-group-append">
+                </span>
+                <span data-search-clear="">
+                  <i class="fas fa-times"></i>
+                </span>
+              </div>
+            </form>
+            <a class="nav-link m-0 p-0" href="https://github.com/fsprojects/fantomas">GitHub<i
+                class="bi bi-box-arrow-up-right mx-2"></i></a>
+          </div>
         </div>
-        <ul class="list-unstyled ps-0" id="sidebar-generation">
-            {{fsdocs-list-of-documents}}
-            <!-- {{fsdocs-list-of-namespaces}} -->
-        </ul>
+      </nav>
+    <div class="container p-0">
+      <div id="fsdocs-content">
+        {{fsdocs-content}}
+        {{fsdocs-tooltips}}
+      </div>
+      </div>
     </div>
-    <div class="container">
-        <div class="masthead">
-            <h3 class="muted"><a href="{{fsdocs-collection-name-link}}">{{fsdocs-collection-name}}</a></h3>
-        </div>
-        <hr/>
-        <div class="container" id="fsdocs-content">
-            {{fsdocs-content}}
-            {{fsdocs-tooltips}}
-        </div>
     </div>
-</main>
-<script type="module" src="{{root}}content/sidebar-generation.js"></script>
-<!-- BEGIN SEARCH BOX: this adds support for the search box -->
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.css"/>
-<script type="text/javascript">var fsdocs_search_baseurl = '{{root}}';</script>
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/lunr.js/2.3.8/lunr.min.js"></script>
-<script type="text/javascript"
-        src="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.min.js"></script>
-<script type="text/javascript" src="{{root}}content/fsdocs-search.js"></script>
-<!-- END SEARCH BOX: this adds support for the search box -->
-<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
-        integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
-        crossorigin="anonymous"></script>
+    
+  </main>
+  <script type="module" src="{{root}}content/sidebar-generation.js"></script>
+  <script type="text/javascript">
+var sidebar = document.getElementById('sidebar-wrapper')
```

Comment:
`var` is a thing of the past. You now mostly want to use `const` or `let` (when variable is mutable).